tests/grid/grid_tools_aspect_ratio.cc

```diff
@@ -0,0 +1,188 @@
+#include <deal.II/base/exceptions.h>
```

Comment:
Can you go to another test file and copy the standard deal.II header (copyright notice etc.) in here?

Reply:
done in https://github.com/dealii/dealii/pull/8894/commits/64e29053cb6ebb238940ffda402e6f043de45d51